src/Arcus.WebApi.Correlation/IServiceCollectionExtensions.cs

```diff
@@ -10,35 +12,21 @@ namespace Arcus.WebApi.Correlation
     // ReSharper disable once InconsistentNaming
     public static class IServiceCollectionExtensions
     {
-        /// <summary>
-        /// Adds operation and transaction correlation to the application.
-        /// </summary>
-        /// <param name="services">The services collection containing the dependency injection services.</param>
-        public static IServiceCollection AddCorrelation(this IServiceCollection services)
-        {
-            Guard.NotNull(services, nameof(services));
-
-            return AddCorrelation(services, configureOptions: null);
-        }
-
         /// <summary>
         /// Adds operation and transaction correlation to the application.
         /// </summary>
         /// <param name="services">The services collection containing the dependency injection services.</param>
         /// <param name="configureOptions">The function to configure additional options how the correlation works.</param>
-        public static IServiceCollection AddCorrelation(
+        public static IServiceCollection AddHttpCorrelation(
```

Comment:
Let's stick with what we had
```suggestion
        public static IServiceCollection AddCorrelation(
```

Reply:
But I can see the point of it. If we use this name, we have to provide the old one and deprecate it.

Reply:
It was actually for making things maybe more clear.
The `Arcus.Observability.Observability` already defines an `.AddCorrelation` so as developer, an ambiguous method is presented... 😞 .

Reply:
Then I would deprecate current one and introduce the new one next to it

Reply:
Done.